src/math/Ray.js

```diff
@@ -228,14 +228,14 @@ class Ray {
 
 		const thc = Math.sqrt( radius2 - d2 );
 
-		// t0 = first intersect point - entrance on front of sphere
```

Comment:
TBH, I find it more readable if `t0` is computed first (since it's the first intersect point) and `t1` after that. The additional check in the `if` statement should not be noticeable performance-wise.

Reply:
I can just remove the check without moving this line.